docs/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ Version 3.42.0-eisop4 (April ?, 2024)
 
 **Implementation details:**
 
+Deprecated the `GenericAnnotatedTypeFactory#addComputedTypeAnnotations(Tree, AnnotatedTypeMirror, boolean)` overload.
+Instead, set `GenericAnnotatedTypeFactory#useFlow` appropriately and
```

Comment:
Couldn't we make `addComputedTypeAnnotationsWithoutFlow` protected and final to allow uses in subclasses but prevent overrides?

Reply:
If you do this, also mention the method in the manual.

Reply:
Made it as final and protecetd. Also add some description in the manual.

Reply:
I also see deletions in `creating-a-checker.tex`, no additions. Did you forget to add your changes in the manual?